components/admin_console/user_autocomplete_setting/user_autocomplete_setting.tsx

```diff
@@ -27,8 +27,8 @@ type Props = {
 const UserAutocompleteSetting = ({id, label, placeholder, helpText, value, onChange, disabled, actions}: Props) => {
     const userSuggestionProvidersRef = useRef([new GenericUserProvider(actions.autocompleteUsers)]);
 
-    const handleChange = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
-        onChange(id, e.target.value);
+    const handleChange = useCallback((value: string) => {
```

Comment:
The thought here is to allow a bit more expandability in the lower level components. Right now it's passing up an onChange event which is harder to control, and requires some hacking for the components that don't send it this way. So, instead my thought is changing it to a string so you always know the proper incoming value to it and leave it up to the lower level component to transcribe it from an onChange event to a string. 

Future thought, this makes it easier for a WYSIWYG editor because then we can just pass up/down a markdown string.